spec/models/users/data_request_spec.rb

```diff
@@ -0,0 +1,41 @@
+# == Schema Information
+#
+# Table name: users_data_requests
+#
+#  id         :bigint           not null, primary key
+#  request_id :string           not null
+#  status_id  :integer          not null
+#  user_id    :bigint           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+RSpec.describe Users::DataRequest, type: :model do
+  context 'validations' do
+    it 'is invalid without a request_id' do
+      data_request = build(:empty_request_id_data_request)
+      expect(data_request).to have(1).error_on(:request_id) 
+    end
+
+    it 'is invalid without a status_id' do
+      data_request = build(:partial_data_request)
+      expect(data_request).to have(2).error_on(:status_id) 
```

Comment:
What are there two errors?

Reply:
1. presence.
2. inclusion.
Both are treated separatedly. There are 2 validations on status_id. `nil` is blank and also not amongst the included values.